pudb/forked.py

```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import absolute_import, print_function, unicode_literals
+
+
+import sys
+import fcntl
+import termios
+import struct
+
+from pudb.debugger import Debugger
+
+
+def forked_pudb():
+    frame = sys._getframe().f_back
+    try:
+        # Getting terminal size
+        s = struct.unpack(
+            "hh",
+            fcntl.ioctl(1, termios.TIOCGWINSZ, "1234"),
+        )
+        term_size = (s[1], s[0])
+    except Exception:
+        term_size = (80, 24)
+
+    Debugger(
+        stdin=open("/dev/stdin"),
+        stdout=open("/dev/stdout", "w"),
```

Comment:
Linux-specific?

Reply:
Yes, absolutely. Unfortunately I don't have much experience on Windows. I guess it should work on MacOS, too, but can't test it myself (same for Windows). I'll add it to the documention.

Reply:
I'm not sure if PuDB works at all on Windows. We should make it work on Mac, though.

Reply:
@asmeurer Do you have a though on how we could achieve the same thing on a Mac?

Reply:
I'd assume this works on MacOS as-is, `/dev/stdin` and `/dev/stdout` are valid there, too.

Reply:
OIC. LGTM then.